loqusdb/commands/delete.py

```diff
@@ -54,7 +54,7 @@ def delete(ctx, family_file, family_type, case_id):
     existing_case = adapter.case({'case_id': case_id})
     if not existing_case:
         LOG.warning("Case %s does not exist in database" %case_id)
-        context.abort
+        ctx.abort()
```

Comment:
```suggestion
        return
```
I think it's better to exit with 0 than 1 when nothing went wrong. I know this is not you changing anything!